src/constants/fields/drugremsspl.yaml

```diff
@@ -0,0 +1,806 @@
+properties:
```

Comment:
@violetcrestedwren I had to make a bunch of schema changes when addressing Ed's feedback. We will need to sync this field list with the latest `schemas/remsspl_mapping.json`